pkgs/top-level/all-packages.nix

```diff
@@ -13094,7 +13094,6 @@ with pkgs;
     emacs30-nox
     emacs30-pgtk
 
-    emacs28-macport
```

Comment:
In addition, removing packages need to add release notes and package alias.  See my comment [here](https://github.com/NixOS/nixpkgs/pull/393512#discussion_r2020332891).